src/Aspire.Dashboard/wwwroot/css/app.css

```diff
@@ -53,6 +55,10 @@ fluent-toolbar[orientation=horizontal] {
     --reconnection-ui-bg: #D6D6D6;
     --error-counter-badge-foreground-color: #ffffff;
 
+    /* overrides of default fluentui-blazor styling */
+    --error: #E10B11 !important;
+    --disabled-opacity: 0.85 !important;
```

Comment:
Setting this variable is a global change. Is it possible to limit the scope?

Reply:
I haven't found a way to do that, considering fluentui-blazor applies the color directly to the elements, and I don't think any selector is specific enough to override a style applied directly to an element?